brainstormforce/ultimate-addons-for-gutenberg
# Fix: Typography - Default string added multiple time in font weight after slider scroll.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/0ce4e760-9da1-4bd3-a275-1b6d44711c24?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- https://share.getcloudapp.com/E0ug02k6

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop any block. for e.g Spectra Heading block
- Inspect the heading first and check default weight should not applying to heading.
- Then Go to Style > Typography for heading > Click on Font size responsive and check if the Default string getting added in the weight dropdown or not.
- Try switching the responsive setting for font size, Changing font size on the responsive device, Default should remain the same It should not duplicate inside the Font-Weight dropdown.

### Checklist:
